opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -60,6 +61,7 @@ NEW_PROP_TAG(RegularizationFactorMsw);
 SET_SCALAR_PROP(FlowModelParameters, DbhpMaxRel, 1.0);
 SET_SCALAR_PROP(FlowModelParameters, DwellFractionMax, 0.2);
 SET_SCALAR_PROP(FlowModelParameters, MaxResidualAllowed, 1e7);
+SET_SCALAR_PROP(FlowModelParameters, RelaxedMaxPvFraction, .03);
 SET_SCALAR_PROP(FlowModelParameters, ToleranceMb, 1e-6);
 SET_SCALAR_PROP(FlowModelParameters, ToleranceCnv,1e-2);
 SET_SCALAR_PROP(FlowModelParameters, ToleranceCnvRelaxed, 1e9);
```

Comment:
I suggest to change the default ToleranceCnvRelaxed to 1 or 10. Even for small cells we should care a bit about the convergence. Can you test how changing the relaxed tolerance change performance and results.

Reply:
I suggest changing RelaxedMaxPvFraction to .0. Then it shouldn't impact anything by default. Then merge the code. A merged code will simplify the testing.

Reply:
RelaxedMaxPvFraction is now zero.